packages/addon-rslib/package.json

```
@@ -32,15 +32,15 @@
   },
   "devDependencies": {
     "@rsbuild/core": "^1.1.12",
-    "@rslib/core": "^0.2.0",
+    "@rslib/core": "^0.2.2",
     "@types/node": "^18.0.0",
     "storybook": "8.5.0-alpha.22",
     "storybook-builder-rsbuild": "workspace:*",
     "typescript": "^5.7.2"
   },
   "peerDependencies": {
     "@rsbuild/core": "^1.0.1",
-    "@rslib/core": ">= 0.1.1",
+    "@rslib/core": "~0.2.2",
```

```suggestion
    "@rslib/core": ">= 0.1.1 || >=0.2",
```

What about this, as a variant of current implementation?

---

seems like it also works so I used it. It's quite unambiguous but at least it will satisfy NPM I hope. 

I think the problem is happening because of `0` major version in general

---

Yes, sadly we need to add `|| >= 0.3` when 0.3 released 😅. Let's merge it.

---

Thank you.